samples/net/cobs_throughput_client/src/main.c

```diff
@@ -0,0 +1,439 @@
+	.server_port = SERVER_PORT,
+	.packet_size = CONFIG_SLIP_THROUGHPUT_PACKET_SIZE,
+	.duration_sec = CONFIG_SLIP_THROUGHPUT_DURATION_SEC,
+	.delay_ms = 0,
+	.running = false,
+};
+
+/* Statistics */
+static struct {
+	uint64_t packets_sent;
+	uint64_t packets_received;
+	uint64_t bytes_sent;
+	uint64_t bytes_received;
+	uint64_t errors;
+	uint64_t pattern_errors;
+	uint64_t timeouts;
+	uint32_t min_rtt_us;
+	uint32_t max_rtt_us;
+	uint64_t total_rtt_us;
+	uint32_t start_time;
+} stats;
+
+/* Fill packet with predictable pattern */
+static void fill_pattern(uint8_t *data, size_t len, uint32_t seq)
+{
+	/* Pattern: sequence number (4 bytes) + repeating byte pattern */
+	memcpy(data, &seq, sizeof(uint32_t));
+	
+	for (size_t i = sizeof(uint32_t); i < len; i++) {
+		data[i] = (uint8_t)((seq + i) & 0xFF);
+	}
+}
+
+/* Verify received packet pattern */
+static bool verify_pattern(const uint8_t *data, size_t len, uint32_t seq)
+{
+	if (len < sizeof(uint32_t)) {
+		return false;
+	}
+
+	uint32_t pkt_seq;
+	memcpy(&pkt_seq, data, sizeof(uint32_t));
+	
+	if (pkt_seq != seq) {
+		LOG_WRN("Sequence mismatch: expected %u, got %u", seq, pkt_seq);
+		return false;
+	}
+
+	/* Verify the pattern */
+	for (size_t i = sizeof(uint32_t); i < len; i++) {
+		uint8_t expected = (uint8_t)((seq + i) & 0xFF);
+		if (data[i] != expected) {
+			LOG_WRN("Pattern error at offset %zu: expected 0x%02x, got 0x%02x",
+				i, expected, data[i]);
+			return false;
+		}
+	}
+
+	return true;
+}
+
+/* Print final statistics */
+static void print_final_stats(void)
+{
+	uint32_t elapsed = k_uptime_get_32() - stats.start_time;
+	uint32_t elapsed_sec = elapsed / 1000;
+
+	if (elapsed_sec == 0) {
+		elapsed_sec = 1;
+	}
+
+	uint64_t tx_kbps = (stats.bytes_sent * 8) / (elapsed_sec * 1000);
+	uint64_t rx_kbps = (stats.bytes_received * 8) / (elapsed_sec * 1000);
+	uint32_t avg_rtt_us = stats.packets_received > 0 ?
+			      (uint32_t)(stats.total_rtt_us / stats.packets_received) : 0;
+
+	LOG_INF("");
+	LOG_INF("=== Final Statistics ===");
+	LOG_INF("Test duration: %u seconds", elapsed_sec);
+	LOG_INF("Packet size: %u bytes", test_params.packet_size);
+	LOG_INF("");
+	LOG_INF("TX: %llu packets, %llu bytes (%llu kbps)",
+		stats.packets_sent, stats.bytes_sent, tx_kbps);
+	LOG_INF("RX: %llu packets, %llu bytes (%llu kbps)",
+		stats.packets_received, stats.bytes_received, rx_kbps);
+	LOG_INF("");
+	LOG_INF("Packet loss: %llu (%.2f%%)",
+		stats.packets_sent - stats.packets_received,
+		stats.packets_sent > 0 ? 
+		(double)(stats.packets_sent - stats.packets_received) * 100.0 / stats.packets_sent : 0.0);
+	LOG_INF("Errors: %llu, Pattern errors: %llu, Timeouts: %llu",
+		stats.errors, stats.pattern_errors, stats.timeouts);
+	LOG_INF("");
+	LOG_INF("RTT (us): min=%u, avg=%u, max=%u",
+		stats.min_rtt_us, avg_rtt_us, stats.max_rtt_us);
+	LOG_INF("======================");
+}
+
+/* Run throughput test */
+static int run_test(void)
+{
+	int sock;
+	struct sockaddr_in server_addr;
```

Comment:
We should not mix POSIX and network namespaced APIs any more, this was changed in commit a63dbfb6cc401f0c22408ba0ac0253b2a1137d44. As this sample does not really need to use POSIX symbols, I suggest you can `CONFIG_NET_NAMESPACE_COMPAT_MODE=n` to catch these in these samples.